CHANGELOG.md

```diff
@@ -4,6 +4,35 @@ All notable changes to this project will be documented in this file.
  
 The format is based on [Keep a Changelog](http://keepachangelog.com/).
 
+## [0.5.0] - 2023-04-13
```

Comment:
Because this release contains new features, the version number resolver suggests version `0.5.0`, not `0.4.2`